lambdas/utils/lloyd_george_validator.py

```diff
@@ -167,7 +168,7 @@ def validate_with_pds_service(file_name_list: list[str], nhs_number: str):
         if patient_details.general_practice_ods != current_user_ods:
             raise LGInvalidFilesException("User is not allowed to access patient")
 
-    except (ValidationError, ClientError, ValueError) as e:
+    except (ClientError, ValueError) as e:
```

Comment:
What is the reason for removing `ValidationError`?

Reply:
ValidationError inherits from ValueError, so it'll be caught by this either way